sources/core/Stride.Core/PlatformType.cs

```diff
@@ -59,13 +59,6 @@ public enum PlatformType
         /// <summary>
         /// macOS
         /// </summary>
-        macOS,
-#pragma warning restore SA1300 // Element must begin with upper-case letter
-
-        /// <summary>
-        /// The Universal Windows Platform (UWP). Please use <see cref="UWP"/> intead.
-        /// </summary>
-        [Obsolete("Please use UWP instead")]
-        Windows10 = UWP,
+        macOS
```

Comment:
Nitpick: you should leave a trailing comma in case someone adds another value to the enum and git doesn't need to touch the last value for the diff

Reply:
Fixed